buildspecs/j9.flags

```diff
@@ -1858,6 +1858,13 @@ Only available on zOS</description>
 			<require flag="interp_jniSupport"/>
 		</requires>
 	</flag>
+	<flag id="opt_shrMsyncSupport">
```

Comment:
Currently this is enabled on z/OS only. Not sure about the future. I prefer to keep this flag for now just in case a second platform need to enable this in the future.